internal/clients/team6/iigo.go

```diff
@@ -58,17 +58,17 @@ func (c *client) CommonPoolResourceRequest() shared.Resources {
 	cprLeft := c.ServerReadHandle.GetGameState().CommonPool
 	reqResource := shared.Resources(3.0 * livingCost)
 
-	defer c.Logf("Request %v from common pool", reqResource)
-	//when common pool does not have enough resource, will not request
-
-	if ourStatus == shared.Critical {
-		return minThreshold - ourResources
+	if ourStatus == shared.Critical && ourResources < minThreshold {
+		return minThreshold - ourResources + shared.Resources(0.01)
```

Comment:
why 0.01? 0

Reply:
so == minimumThreshold will not count as critical status right?

Reply:
ah. I see what you did there,
https://github.com/SOMAS2020/SOMAS2020/blob/73ed812b5add9f854209267fa1d66285f8a1199a/internal/server/helper.go#L63

you dont need that extra 0.01 to get out of critical

Reply:
thx for this!